Update automation-runbook-types.md

sckissel opened the pull request:
Removed duplicate "cannot" in "PowerShell runbooks cannot cannot retrieve an unencrypted [variable asset](automation-variables.md) with a null value" line

azurecla commented:
Hi __@joseidz__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (joseidz). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

sckissel committed:
Update automation-runbook-types.md

Removed duplicate "cannot" in "PowerShell runbooks cannot cannot retrieve an unencrypted [variable asset](automation-variables.md) with a null value" line
